From 85b2a60ff9d8db8c52fa806e74f4f06a644095a2 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Thu, 12 Jan 2023 15:32:32 +0200
Subject: [PATCH 37/63] dts/freescale: s32cc: Make STM7 a timestamping counter

STM7 is used as a timestamping source for CAN modules.

Issue: ALB-9261
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/s32cc.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/s32cc.dtsi b/arch/arm64/boot/dts/freescale/s32cc.dtsi
index 7f73c92a668c..d3302ebd2a4d 100644
--- a/arch/arm64/boot/dts/freescale/s32cc.dtsi
+++ b/arch/arm64/boot/dts/freescale/s32cc.dtsi
@@ -678,7 +678,7 @@ stm6: stm@40224000 {
 		};
 
 		stm7: stm@40228000 {
-			compatible = "nxp,s32cc-stm-global";
+			compatible = "nxp,s32cc-stm-ts";
 			reg = <0x0 0x40228000 0x0 0x3000>;
 			clocks = <&clks S32CC_SCMI_CLK_STM_MODULE>;
 			clock-names = "stm";
-- 
2.25.1

